collect_app/src/main/java/org/odk/collect/android/utilities/ColorPickerDialog.java

```diff
@@ -392,6 +394,9 @@ public ColorPickerDialog(Context context, OnColorChangedListener listener,
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        scaleFactor = (int) (getContext().getResources().getDisplayMetrics().heightPixels * 0.9 / 366);
+
+
```

Comment:
If you think a blank line is needed to make the code clearer that's ok but one is enough not two.